views/scss/item-creator.scss

```diff
@@ -789,6 +789,7 @@ img.dragging.ui-draggable-dragging {
         .widget-box {
             &.edit-sleep {
                 position: relative;
+                width: 100% !important;
```

Comment:
That will break a `Size and positioning` feature in authoring mode, won't it?
![image](https://user-images.githubusercontent.com/2943256/88026843-b98e3a80-cb36-11ea-9008-5012371c6774.png)

Reply:
You tested in your local?

Reply:
I'm asking you to kindly check that if it hasn't been done already.

Reply:
As I said in DSM, from the list of all possible solutions I discover, this is the best. Don't break animation but still fix the bug when you click outside and works for `items` and `mediaManager`. If you see some unexpected behavior tell me

Reply:
Well, the `span` itself then stretches out, as it shouldn't, and the picture changes size back and forth when you click on it.

Try the following instead.
```suggestion

                img {
                    min-width: 100%;
                }

```